UnityProject/Assets/Scripts/Items/Dice/RollDie.cs

```diff
@@ -134,6 +134,11 @@ private void ThrowStart(UniversalObjectPhysics throwInfo)
 			Chat.AddActionMsgToChat(throwInfo.thrownBy, $"You throw the {dieName}...", $"{throwInfo.thrownBy.ExpensiveName()} throws the {dieName}...");
 		}
 
+		private void ThrowStartEndold(UniversalObjectPhysics throwInfo)
+		{
+			this.RestartCoroutine(WaitForSide(), ref waitForSide);
```

Comment:
I don't understand why is this method needed now. Did something break after the change to new physics?

I also can't read the name of the method.

Reply:
throwing works differently now so you can't use the old system exactly

Reply:
ok so why does it need a new method instead of using the one we have now? Is it too hard to make rolling dice use the new signature instead of creating a ``ThrowEndOld`` method?

Reply:
well, you can't reuse it because it expects there to be momentum difference but, implicitly by the name it saying that it's finished moving, so therefore there is no momentum change

Reply:
momentum 0 ?

Reply:
yeah, because it stopped moving, Because it's reached the end of the throw

Reply:
yes, so why not call ``OnThrowEnd`` with ``0`` as the argument

Reply:
actually, why does ThrowEnd need a force if it is called when it has no force?

Reply:
That's what uses the old method because it doesn't have a force, and also loads of other stuff still uses it